### ISSUE TITLE
Fixes for audio_recognition

### ISSUE THREAD

**Issue by ojotoxy** [latentai/model-zoo-models#8]
- (Start by pulling in changes, we did some formatting changes to the code & readme.)
- None of the compile example command lines work, please fix
- Please upload trained tf checkpoint to s3 using "zoo-hash-upload" and add the weights files to models_index.json in the root of the repo. (Look at vgg16 as an example - although it looks like you only are saving the tf checkpoint here, not the keras model.
- Document how to download the trained model with "leip zoo download --model_id .... --variant_id ..." and use it for evaluation.

**Comment by bohdankornienko** [comment]
Please see https://github.com/latentai/model-zoo-models/pull/9

**Comment by ojotoxy** [comment]
Looks like the only issue remaining is getting the compile examples to work.

**Comment by bohdankornienko** [comment]
Yes. And it should be fixed automatically once sdk will support automatic dropout removal.
